docs/architecture/adr-006-advertise-contract.md

```diff
@@ -0,0 +1,113 @@
+# ADR 006: Advertise contract
+
+## Changelog
+
+- Oct 20, 2022: Initial draft;
+
+## Status
+
+DRAFT
+
+## Abstract
+This ADR defines the architecture of the Advertise contract. This contract allows users to advertise a post, then social network applications can query random advertisement posts from it.
```

Comment:
```suggestion
This ADR defines the architecture of the Advertisement contract. This contract allows users to advertise a post so that later any social network application can query random advertised posts from it.
```